web-api/src/embedding.rs

```diff
@@ -38,7 +38,8 @@ pub(crate) struct Embedder {
 }
 
 impl Embedder {
-    pub(crate) fn load(config: &Config) -> Result<Self, SetupError> {
+    #[allow(clippy::unused_async)]
```

Comment:
:art: it would be good to have a comment at this lint like
```suggestion
    #[allow(clippy::unused_async)] // required for sagemaker
```